start-flask.sh

```diff
@@ -5,4 +5,5 @@ if ! [ -f endpoints.yml ]; then
   cp endpoints.yml.template endpoints.yml
 fi
 
-exec python panoptes_aggregation/routes.py
\ No newline at end of file
+LISTEN_PORT=${LISTEN_PORT:=80}
+exec gunicorn -b 0.0.0.0:$LISTEN_PORT "panoptes_aggregation.routes:make_application()"
```

Comment:
Do we want to listen to all ipv4 interfaces? What about IP V6? Is there a ENV var we can use here to listen to a specific interface?

Reply:
Do we want to set the number of workers to handle incoming requests? http://docs.gunicorn.org/en/stable/faq.html#faq

http://docs.gunicorn.org/en/stable/settings.html#worker-processes

I'm not sure how well that will play with the global vars i described in https://github.com/zooniverse/aggregation-for-caesar/issues/166 so would need some testing....

Reply:
We do want to listen on all interfaces – Docker/Kubernetes create virtual interfaces and only the correct one is visible to the container. Our instances don't have IPv6 at the moment.

Reply:
It might be worth setting the number of workers though, since this may get a decent amount of traffic. I'd suggest maybe setting it to 4?

Reply:
I can set the port to be `-b "[::]:$LISTEN_PORT"` and it will listen on both ipv4 and ipv6.  Would that work or should we still set it up as an ENV var?

As for the number of workers I think @adammcmaster would have a better idea.